lms/djangoapps/certificates/management/commands/rescind_cert.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to update the status of a single user certificate
+for a given user, in a given course. If no --status kwarg is provided,
+the status will be updated to CertificiateStatuses.unavailable.
+"""
+import logging
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+
+from certificates.models import GeneratedCertificate, CertificateStatuses
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    '''Update the status of a single user certificate for a given user in a given course'''
+    help = u'Rescind a certificate for a particular user in a particular course.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-c', '--course',
```

Comment:
For consistency, I'd like to see us use the same term throughout `course_id` vs `course`;
my preference is the former.

Consistency aside, it's more explicit about the data format, which is good.

If we do that, we can presumably drop the `metavar`/`dest` kwargs here.
Though, you'll just need to update the `course` variable to `course_id` in the `handle` method.